randomScripts/Cards_PostProcessing/creatJobs.py

```diff
@@ -1,5 +1,10 @@
 import os
+import argparse
 
+parser = argparse.ArgumentParser(description='Creat Jobs')
+parser.add_argument('Directory', type=str, help='The directory we are processing.')
+parser.add_argument('Region', type=str, help='The region we are processing.')
```

Comment:
What about also adding an argment to enable UL if wanted (can be off by default if you want) instead of uncommenting down?